src/Gui/EditableDatumLabel.cpp

```diff
@@ -129,9 +137,7 @@ void EditableDatumLabel::deactivate()
         cameraSensor = nullptr;
     }
 
-    if (viewer) {
-        static_cast<SoSeparator*>(viewer->getSceneGraph())->removeChild(root); // NOLINT
-    }
+    root->whichChild = -1;
```

Comment:
```suggestion
    root->whichChild = SoSwitch::SO_SWITCH_NONE;
```

Would be better here as it would be properly labelled. I have not compiles it though.